docs/observation/payload.md

```diff
@@ -1,34 +1,52 @@
 # Format of observation payload
 
-Notifications received by a source repository, such as GitHub, OneDrive, Google Drive, etc. have a provider specific format, e.g. OneDrive uses the format described on Microsoft's [docs](https://docs.microsoft.com/en-us/graph/api/driveitem-delta?view=graph-rest-1.0&tabs=http) site. When those notifications are received in the `onedrive-change-listener`, they get converted to a portable format before sent to the next stage, e.g.:
+Notifications received by a source repository, such as GitHub, OneDrive, Google Drive, etc. have a provider specific format, e.g. OneDrive uses the format described on Microsoft's [docs](https://docs.microsoft.com/en-us/graph/api/driveitem-delta?view=graph-rest-1.0&tabs=http) site. When those notifications are received in the respective listener (e.g. `onedrive-change-listener` for OneDrive), they get converted to a portable format before sent to the next stage, e.g.:
 
 ```
 {
   "type": "onedrive",
-  "provider" {
-    "driveId": "..."
-  },
   "owner": "me",
   "repo": "myrepo",
   "ref": "master",
-  "mountpoint": {
-    "path": "/ms/",
-    "root": "/myshare"
-  },
   "changes": [
     ...
   ],
+  "mountpoint": {
+    "path": "/ms/",
+    "root": "/myshare"
+  }
 }
 ```
-The `provider` section contains provider specific information that might be required for downstream handlers.
+The `changes` array contains an entry per change seen since the last batch of changes was sent, e.g. for OneDrive this might look as follows:
+
+```
+{
+  "path": "/myshare/doc1",
+  "time": "2020-03-03T20:05:23Z",
+  "type": "modified",
```

Comment:
maybe adding a section that explains the changes in more detail (eg table of properties) and also enumerates the possible values.